.github/workflows/buildAll.yml

```diff
@@ -1,5 +1,17 @@
 name: Build All
 on:
+  schedule:
+    - cron: 15 6 * * *
+  workflow_dispatch:
+    inputs:
+      branch:
+        type: string
+        required: false
+        default: 'develop'
+      label:
+        type: string
+        required: false
+        default: 'VS Code Extensions'
```

Comment:
Instead of overloading the buildAll Action let's create a separate `nightlyE2EDevelop.yml` file where we have the cron and the two jobs: buildAll & e2e-tests

This action should have one job.